core/src/test/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/upgrade/ccm/RemoveAgentHandlerTest.java

```diff
@@ -51,31 +51,30 @@ void selector() {
 
     @Test
     void doAccept() throws CloudbreakOrchestratorException {
-        UpgradeCcmRemoveAgentRequest request = new UpgradeCcmRemoveAgentRequest(STACK_ID, CLUSTER_ID, Tunnel.CCM);
+        UpgradeCcmRemoveAgentRequest request = new UpgradeCcmRemoveAgentRequest(STACK_ID, CLUSTER_ID, Tunnel.CCM, null);
         when(event.getData()).thenReturn(request);
 
         Selectable result = underTest.doAccept(event);
         InOrder inOrder = inOrder(upgradeCcmService);
-        inOrder.verify(upgradeCcmService).updateTunnel(STACK_ID);
+        inOrder.verify(upgradeCcmService).updateTunnel(STACK_ID, Tunnel.latestUpgradeTarget());
         inOrder.verify(upgradeCcmService).removeAgent(STACK_ID, Tunnel.CCM);
+        assertThat(((UpgradeCcmRemoveAgentResult) result).getAgentDeletionSucceed().equals(Boolean.FALSE));
         assertThat(result.selector()).isEqualTo("UPGRADECCMREMOVEAGENTRESULT");
     }
 
     @Test
     void orchestrationException() throws CloudbreakOrchestratorException {
```

Comment:
Please also verify `upgradeCcmService.removeAgentFailed()` and `((UpgradeCcmRemoveAgentResult) result).getAgentDeletionSucceed() == TRUE`.

Reply:
ok